examples/llama2/llama2_genai.py

```diff
@@ -0,0 +1,60 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+# --------------------------------------------------------------------------
+
+import argparse
+import json
+
+import olive.workflows.run as olive_run
```

Comment:
can you import this as `from olive.workflows import run as olive_run` to avoid the codeql error?

Reply:
Done!